### ISSUE TITLE
Console Error: Access to XMLHttpRequest at [...]

### ISSUE THREAD

**Issue by JamieArmstrongArndt** [hochschule-darmstadt/openartbrowser#163]
A console error is thrown when entering [a movement page on the staging server](https://cai-artbrowserstaging.fbi.h-da.de/de/movement/Q14378):

```stacktrace
Access to XMLHttpRequest at 'data:text;charset=utf-8,' (redirected from
'https://googleads.g.doubleclick.net/pagead/id') from origin 'null' has 
been blocked by CORS policy: No 'Access-Control-Allow-Origin' header 
is present on the requested resource. 
```

**Comment by lbucas** [comment]
I was not able to reproduce that. I think this occurs because we don't use youtube in privacy mode yet. I will open an issue.

**Comment by lbucas** [comment]
Should be resolved with #164
